decidim-conferences/app/cells/decidim/conferences/conference_m/show.erb

```diff
@@ -0,0 +1,13 @@
+<%= link_to resource_path, class: "conference__list__item" do %>
```

Comment:
On narrower mobile screens these can become quite cluttered when we have 2 per row:
![Conferences mobile](https://user-images.githubusercontent.com/864340/187423752-f4a027c9-0ef7-4cc4-af46-39ab58ce9d14.png)

Maybe we should just have one item per row?